packages/tools/kadena-cli/src/wallets/commands/walletImport.ts

```diff
@@ -44,16 +44,20 @@ export const createImportWalletCommand: (
       });
```

Comment:
did we want to add an extra prompt? 
> first show mnemonic phrase in output
> prompt `did you save the mnemonic phrase in a safe and secure place (Y/N): `
> continue with command as usual

Reply:
Sure, we might entertain the idea of doing just that, however, given the astonishingly high level of the Extreme Indicator, one would naturally assume, nay, confidently expect, that they would possess the intellectual wherewithal — indeed, an almost uncanny level of acumen — to not only grasp it, but they would intuitively, almost instinctively know that they should, without any shadow of a doubt, be smart enough to securely store it :P

What do you think?

Reply:
🤷‍♂️